test/unit/argument_parser/detail/format_man_test.cpp

```diff
@@ -0,0 +1,237 @@
+                         ".SH NAME\n"
+                         "default \\- A short description here.\n"
+                         ".SH SYNOPSIS\n"
+                         "\\fB./format_man_test\\fP synopsis\n"
+                         ".br\n"
+                         "\\fB./format_man_test\\fP synopsis2\n"
+                         ".SH DESCRIPTION\n"
+                         "description\n"
+                         ".sp\n"
+                         "description2\n"
+                         ".SH POSITIONAL ARGUMENTS\n"
+                         ".TP\n"
+                         "\\fBARGUMENT-1\\fP (\\fIsigned 8 bit integer\\fP)\n"
+                         "this is a positional option. \n"
+                         ".TP\n"
+                         "\\fBARGUMENT-2\\fP (\\fIList\\fP of \\fIstd::string\\fP's)\n"
+                         "this is a positional option. Default: []. \n"
+                         ".SH OPTIONS\n"
+                         ".SS Basic options:\n"
+                         ".TP\n"
+                         "\\fB-h\\fP, \\fB--help\\fP\n"
+                         "Prints the help page.\n"
+                         ".TP\n"
+                         "\\fB-hh\\fP, \\fB--advanced-help\\fP\n"
+                         "Prints the help page including advanced options.\n"
+                         ".TP\n"
+                         "\\fB--version\\fP\n"
+                         "Prints the version information.\n"
+                         ".TP\n"
+                         "\\fB--copyright\\fP\n"
+                         "Prints the copyright/license information.\n"
+                         ".TP\n"
+                         "\\fB--export-help\\fP (std::string)\n"
+                         "Export the help page information. Value must be one of [html, man].\n"
+                         ".TP\n"
+                         "\\fB--version-check\\fP (bool)\n"
+                         "Whether to to check for the newest app version. Default: 1.\n"
+                         ".SS \n"
+                         ".TP\n"
+                         "\\fB-i\\fP, \\fB--int\\fP (\\fIsigned 32 bit integer\\fP)\n"
+                         "this is a int option. Default: 5. \n"
+                         ".TP\n"
+                         "\\fB-j\\fP, \\fB--jint\\fP (\\fIsigned 32 bit integer\\fP)\n"
+                         "this is a required int option. \n"
+                         ".SH FLAGS\n"
+                         ".SS SubFlags\n"
+                         "here come all the flags\n"
+                         ".TP\n"
+                         "\\fB-f\\fP, \\fB--flag\\fP\n"
+                         "this is a flag.\n"
+                         ".TP\n"
+                         "\\fB-k\\fP, \\fB--kflag\\fP\n"
+                         "this is a flag.\n"
+                         ".SH EXAMPLES\n"
+                         "example\n"
+                         ".sp\n"
+                         "example2\n"};
```

Comment:
Can you use a raw string literal instead here? I think that would be a lot more readable...